spec/chrono_model/adapter/migrations_spec.rb

```diff
@@ -231,26 +231,38 @@
       before :all do
         adapter.add_index table, [:foo, :bar], :name => 'foobar_index'
         adapter.add_index table, [:test],      :name => 'test_index'
+        adapter.add_index table, :baz
```

Comment:
@gridanjbf add index without name

Reply:
> 

in this case you are adding it to prepare another test, 
i was thinking about adding a test  with an explicit expectation  under describe ".add_index" just for the sake of completeness.

Reply:
got it, will do that